spring-5-webflux/src/main/java/com/baeldung/spring/retry/ExternalConnector.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.spring.retry;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+import reactor.util.retry.Retry;
+
+import java.time.Duration;
+
+@Component
+@AllArgsConstructor
+public class ExternalConnector {
+
+    private static final String PATH_BY_ID = "/data/{id}";
+
+    private final WebClient webClient;
+
+    public Mono<String> getData(String stockId) {
```

Comment:
It would be better if this code used a two space continuation wrap, and if the wider lines broke at the nearest `,` or `.` to enable reading.

Reply:
Reformatted using eclipse formatter (in Intellij). 

The continuation wrap seems to be set to 4 in the eclipse formatter scheme:

<img width="1251" alt="intellij-code-settings" src="https://user-images.githubusercontent.com/994151/115473274-58190a80-a233-11eb-96ba-fb1b521d71e2.png">